$OpenBSD: patch-make.1,v 1.4 2011/04/20 20:51:41 naddy Exp $

Since we install GNU make as gmake replace make with gmake in the
manpage where it makes sense.

--- make.1.orig	Thu Jun  4 08:30:27 2009
+++ make.1	Sat Oct 23 16:23:21 2010
@@ -1,8 +1,8 @@
-.TH MAKE 1 "22 August 1989" "GNU" "LOCAL USER COMMANDS"
+.TH GMAKE 1L "22 August 1989" "GNU" "LOCAL USER COMMANDS"
 .SH NAME
-make \- GNU make utility to maintain groups of programs
+gmake \- GNU make utility to maintain groups of programs
 .SH SYNOPSIS
-.B "make "
+.B "gmake "
 [
 .B \-f
 .I makefile
@@ -18,7 +18,7 @@ which is made from the Texinfo source file
 .SH DESCRIPTION
 .LP
 The purpose of the
-.I make
+.I gmake
 utility is to determine automatically which
 pieces of a large program need to be recompiled, and issue the commands to
 recompile them.
@@ -27,17 +27,17 @@ The manual describes the GNU implementation of
 which was written by Richard Stallman and Roland McGrath, and is
 currently maintained by Paul Smith.
 Our examples show C programs, since they are most common, but you can use
-.I make
+.I gmake
 with any programming language whose compiler can be run with a
 shell command.
 In fact,
-.I make
+.I gmake
 is not limited to programs.
 You can use it to describe any task where some files must be
 updated automatically from others whenever the others change.
 .LP
 To prepare to use
-.IR make ,
+.IR gmake ,
 you must write a file called the
 .I makefile
 that describes the relationships among files in your program, and the
@@ -49,17 +49,17 @@ Once a suitable makefile exists, each time you change 
 this simple shell command:
 .sp 1
 .RS
-.B make
+.B gmake
 .RE
 .sp 1
 suffices to perform all necessary recompilations.
 The
-.I make
+.I gmake
 program uses the makefile data base and the last-modification times
 of the files to decide which of the files need to be updated.
 For each of those files, it issues the commands recorded in the data base.
 .LP
-.I make
+.I gmake
 executes commands in the
 .I makefile
 to update
@@ -71,7 +71,7 @@ is typically a program.
 If no
 .B \-f
 option is present,
-.I make
+.I gmake
 will look for the makefiles
 .IR GNUmakefile ,
 .IR makefile ,
@@ -99,7 +99,7 @@ If
 .I makefile
 is `\-', the standard input is read.
 .LP
-.I make
+.I gmake
 updates a target if it depends on prerequisite files
 that have been modified since the target was last modified,
 or if the target does not exist.
@@ -126,7 +126,7 @@ previous one:
 is equivalent to
 .BR "\-C " /etc.
 This is typically used with recursive invocations of
-.IR make .
+.IR gmake .
 .TP 0.5i
 .B \-d
 Print debugging information in addition to normal processing.
@@ -134,7 +134,7 @@ The debugging information says which files are being c
 remaking, which file-times are being compared and with what results,
 which files actually need to be remade, which implicit rules are
 considered and which are applied---everything interesting about how
-.I make
+.I gmake
 decides what to do.
 .TP 0.5i
 .BI \-\-debug "[=FLAGS]"
@@ -181,7 +181,7 @@ If several
 options are used to specify several directories, the directories are
 searched in the order specified.
 Unlike the arguments to other flags of
-.IR make ,
+.IR gmake ,
 directories given with
 .B \-I
 flags may come directly after the flag:
@@ -203,7 +203,7 @@ option, the last one is effective.
 If the
 .B \-j
 option is given without an argument,
-.IR make
+.IR gmake
 will not limit the number of jobs that can run simultaneously.
 .TP 0.5i
 .BR \-k , " \-\-keep\-going"
@@ -242,7 +242,7 @@ This also prints the version information given by the
 .B \-v
 switch (see below).
 To print the data base without trying to remake any files, use
-.B make
+.B gmake
 .B \-p
 .BI \-f /dev/null.
 .TP 0.5i
@@ -267,11 +267,11 @@ Cancel the effect of the
 .B \-k
 option.
 This is never necessary except in a recursive
-.I make
+.I gmake
 where
 .B \-k
 might be inherited from the top-level
-.I make
+.I gmake
 via MAKEFLAGS or if you set
 .B \-k
 in MAKEFLAGS in your environment.
@@ -281,11 +281,11 @@ Touch files (mark them up to date without really chang
 instead of running their commands.
 This is used to pretend that the commands were done, in order to fool
 future invocations of
-.IR make .
+.IR gmake .
 .TP 0.5i
 .BR \-v , " \-\-version"
 Print the version of the
-.I make
+.I gmake
 program plus a copyright, a list of authors and a notice that there
 is no warranty.
 .TP 0.5i
@@ -294,7 +294,7 @@ Print a message containing the working directory
 before and after other processing.
 This may be useful for tracking down errors from complicated nests of
 recursive
-.I make
+.I gmake
 commands.
 .TP 0.5i
 .B \-\-no\-print\-directory
@@ -314,9 +314,9 @@ Without
 it is almost the same as running a
 .I touch
 command on the given file before running
-.IR make ,
+.IR gmake ,
 except that the modification time is changed only in the imagination of
-.IR make .
+.IR gmake .
 .TP 0.5i
 .B \-\-warn\-undefined\-variables
 Warn when an undefined variable is referenced.
@@ -328,7 +328,7 @@ and no targets that were built failed.  A status of on
 if the
 .B \-q
 flag was used and
-.I make
+.I gmake
 determines that a target needs to be rebuilt.  A status of two will be
 returned if any errors were encountered.
 .SH "SEE ALSO"
